tutorials/21_transform_fbx_to_dae.md

```diff
@@ -0,0 +1,37 @@
+\page transform_fbx_to_dae Transform FBX to Collada in Blender
+
+This tutorial shows how to transform a mesh in fbx format to collada with Blender and change its origin.
+
+You can download Blender +2.81 from [this web site](https://www.blender.org/download/).
+
+# Importing the FBX file
+
+Remove all the elements in the scene. You can select the elements pushing `Ctrl + Shift + left mouse button`
+from the list of elements that you can see in the right top corner of the Blender window.
+
+@image html img/transform_fbx_to_dae/remove_elemets.gif
+
+Then import your FBX model. `File -> Import -> Fbx (.fbx).`
+
+@image html img/transform_fbx_to_dae/import_model.gif
+
+# Modify properties of the model
+
+Now we can edit some of the properties of the object.
+
+## Change the origin of the model
```

Comment:
I just want to confirm that this is an optional step right (and not specific to fbx format)? I think some of our models (usually static background models) have offset on purpose so that when we add the model to the scene, we can just leave the pose at 0 and it'll be at the right location.

If it's an optional step, can we add something like

```
Sometimes a mesh may have an origin at an offset from the desired location. For example, you may want to move the origin of a robot arm to the base or move the origin of a wheel to its center.
```